pallets/compliance-manager/src/benchmarking.rs

```diff
@@ -46,15 +84,32 @@ pub fn make_issuer<T: IdentityConfig + TestUtilsFn<AccountIdOf<T>>>(id: u32) ->
 ///   - It could have more complexity if `TrustedIssuer::trusted_for` is a vector but not on
 ///   benchmarking of add/remove. That could be useful for benchmarking executions/evaluation of
 ///   complience requiriments.
```

Comment:
Any thoughts on this now?

Reply:
That is talking about the cost of executing the compliance requirements during an asset transfer.  Which is out of scope for this PR.